packages/2d/src/components/Txt.ts

```diff
@@ -105,6 +105,28 @@ export class Txt extends Shape {
     context.restore();
   }
 
+  protected override getCacheBBox(): BBox {
+    const cache = super.getCacheBBox();
+
+    if (!this.element.hasChildNodes) return cache;
+
+    const size = this.computedSize();
+    const range = document.createRange();
+    range.selectNodeContents(this.element);
+    const bbox = range.getBoundingClientRect();
+
+    const lineWidth = this.lineWidth();
+    // We take the default value of the miterLimit as 10.
+    const miterLimitCoefficient = 0.5 * 10;
```

Comment:
Also, this coefficient should only be used if `this.lineJoin()` is set to `miter`.
For both `bevel` and `round`, the coefficient can be just `0.5`